### ISSUE TITLE
Develco Siren SIRZB-110

### ISSUE THREAD

**Issue by jarlesb** [dresden-elektronik/deconz-rest-plugin#3350]
## Device

- Product name: Develco Smart Siren
- Manufacturer: Develoc Products A/S
- Model identifier: SIRZB-110
- Device type : 
  - Siren

## Screenshots

[Develco Siren SIRZB-110 - screenshot.pdf](https://github.com/dresden-elektronik/deconz-rest-plugin/files/5322214/Develco.Siren.SIRZB-110.-.screenshot.pdf)
[Develco Siren SIRZB-110 - Tech manual.pdf](https://github.com/dresden-elektronik/deconz-rest-plugin/files/5322216/Develco.Siren.SIRZB-110.-.Tech.manual.pdf)

**Comment by jarlesb** [comment]
Please add device support for the Develco Smart Siren. Currently recognised as light and does not make any noise. See detailed specifications in the attachments.

Thanks!

**Comment by tyr84** [comment]
So, I've added the siren, and it shows up as a light. However, I can only activate the siren through the deconz interface:
![image](https://user-images.githubusercontent.com/27015255/105640281-8463fb80-5e7d-11eb-82b0-14c58ede78cc.png)

I've tried the same method as the develco smoke alarms, setting state-alert to select or lselect, both in the api, and through the deconz.configure service in home assistant, but there is no response from the siren.
![image](https://user-images.githubusercontent.com/27015255/105640366-e7ee2900-5e7d-11eb-8542-3be2c13ae2b4.png)
![image](https://user-images.githubusercontent.com/27015255/105640372-f1779100-5e7d-11eb-9b6a-85b27dc39d35.png)

Am I doing something wrong, also is it possible to reach the deconz functionality shown in the first screenshot through the api? (selecting warning mode etc)
